site/content/docs/5.2/components/close-button.md

```diff
@@ -24,7 +24,7 @@ Disabled close buttons change their `color`. We've also applied `pointer-events:
 
 ## Dark variant
 
-Change the default `.btn-close` to be white with the `.btn-close-white` class.
+Add `.btn-close-white` to the `.btn-close` for a dark variant.
```

Comment:
I'll probably try to propose a change of name for `.btn-close-white` that should talk about "dark" (as dark variant) instead of "white". So I'll try to tackle this change at the same time.
Regarding Sass variable (and CSS variables), it could be an improvement to tackle in another PR yes. I let your create a technical issue about it if you think that's doable.